### ISSUE TITLE
WCAG 2.1 translations missing WCAG 2.0 errata

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag#921]
I am not sure if this is where this issue goes, perhaps AG can forward this to the right people.

In researching the [WCAG 2.0 errata](https://www.w3.org/WAI/WCAG20/errata/) I noticed that the official WCAG 2.1 translations do not appear to have all of the changes outlined in the errata (unlike the English WCAG 2.1 version).

- 1.3.3 added the word "color": Missing from [Danish](https://www.w3.org/Translations/WCAG21-da-CAT/#sensory-characteristics) and [Chinese](https://www.w3.org/Translations/WCAG21-zh/#sensory-characteristics)
- 1.4.3 removed "minimal" from the last bullet: Missing from [Italian](https://www.w3.org/Translations/WCAG21-it/#contrast-minimum) and [Chinese](https://www.w3.org/Translations/WCAG21-zh/#contrast-minimum)

**Comment by cstrobbe** [comment]
Would it still make sense for the working group to published an "edited recommendation" for WCAG 2.1, similar to what the XML Core Working Group did with its editions of XML 1.0? Not only did XML 1.0 go through five editions, but the fifth edition was "modified in place to replace broken links (...)".
(These editions of XML 1.0 exist in parallel with [XML 1.1](https://www.w3.org/TR/2006/REC-xml11-20060816/), which was also edited in place.)

**Comment by patrickhlauke** [comment]
@alastc @iadawn @kfranqueiro can this be closed?
